hw/ip/otp_ctrl/dv/cov/otp_ctrl_cov_excl.el

```diff
@@ -91,3 +90,517 @@ Transition FsmStateError->AccessError "7->5"
 CHECKSUM: "3767637819"
 ANNOTATION: "[EXTERNAL] alert_sender is fully verified by FPV testbench."
 MODULE:prim_diff_decode
+
+//==================================================
+// VCS auto-generated UNR exclusion
+//==================================================
+CHECKSUM: "1713188235 4271210983"
```

Comment:
Hey @msfschaffner below is the auto-generated UNR exclusion file we reviewed this morning.